### ISSUE TITLE
Unknown Error Adding Daikin AC

### ISSUE THREAD

**Issue by aftertommy** [scottyphillips/echonetlite_homeassistant#126]
I am moving my HASS setup over to a Home Assistant Yellow, starting from scratch. 

I'm having some issues adding my air conditioner units, which is strange as they've been working on my previous (docker based) setup. In the attached logs, I'm attempting to add the AC at IP address 192.168.40.127 and another at 192.168.40.121. In the 127 case, I get an "Unknown Error Occurred" error. For the 121 AC, the configuration UI remains unresponsive with a spinner.

For whatever it's worth, I can add _other_ air conditioners to my setup, but these two are giving me issues.
[home-assistant_2023-03-03T12-27-44.625Z.log](https://github.com/scottyphillips/echonetlite_homeassistant/files/10881927/home-assistant_2023-03-03T12-27-44.625Z.log)

**Comment by nao-pon** [comment]
@aftertommy It seems that you need to do a key check for the error in echonetlite/config_flow.py, line 214.

Try changing config_flow.py, line 214 to the following and restarting HA.

```python
         if len(_detected_hosts):
```
to
```python
         if len(_detected_hosts) and self.host in _detected_hosts.keys():
```

**Comment by nao-pon** [comment]
Probably, I think that it was fixed in Version 3.7.3.
